services/nomad/mirror/mirror.nomad

```diff
@@ -92,12 +92,18 @@ server {
                 repo-de.voidlinux.org
                 repo-fi.voidlinux.org
                 repo-us.voidlinux.org
+                repo-fastly.voidlinux.org
                 "~^repo-[a-z]{2}\.voidlinux\.org$";
     root /srv/www;
 
     location / {
         autoindex on;
     }
+
+    location ~* .(?:xbps|sig|iso|gz|xz)$ {
```

Comment:
```
location ~* \.(xbps|sig|iso|gz|xz)$ {
```

or just `~` for non cases sensitive.

Reply:
using a noncapturing group is probably slightly faster